libs.autobuild

```diff
@@ -9,7 +9,9 @@ cmake_package 'knowledge_reasoning/factpp' do |pkg|
     pkg.depends_on 'base/cmake'
 end
 cmake_package 'knowledge_reasoning/owlapi'
-cmake_package 'tools/graph_analysis'
+cmake_package 'tools/graph_analysis' do |pkg|
```

Comment:
Just a suggestion ...

Do the same thing than with Ruby, and provide the package will the full path to the Python interpreter, instead of relying on two different codepaths that resolve the python interpreter independently.

Also, would be good to have a way to disable the Python extension, if it at all makes sense.

Reply:
Regarding the activation see rock-core/package_set#161

What do you mean with the "same thing than with Ruby" - this is not clear to me?

Reply:
Ruby packages should be given the path to the Ruby interpreter as a CMake variable from within Autoproj. This bypasses the package's autodetection mechanism, which in this case is just trying to make sure build will fail in some cases - that's what I meant by "relying on two differnt codepaths that resolve the python interpreter independently"

Example: https://github.com/rock-core/rock-package_set/blob/master/libs.autobuild#L569

Reply:
Ok, now understood. Makes total sense. Updated accordingly.